projects/step-core/src/lib/client/augmented/services/augmented-executions-service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { ExecutionsService } from '../../generated';
+import { TableRestService } from '../../table/services/table-rest.service';
+import { BaseHttpRequest } from '../../generated/core/BaseHttpRequest';
+import { TableRemoteDataSource } from '../../../modules/table/shared/table-remote-data-source';
+import { Observable } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class AugmentedExecutionsService extends ExecutionsService {
+  readonly EXECUTIONS_TABLE_ID = 'executions';
+
+  readonly dataSource = new TableRemoteDataSource(this.EXECUTIONS_TABLE_ID, this._tableRest, {
+    description: 'description',
+    startTime: 'startTime',
+    endTime: 'endTime',
+    user: 'executionParameters.userID',
+    environment: 'executionParameters.customParameters.env',
+    status: 'status',
+    result: 'result',
+  });
+
+  constructor(override httpRequest: BaseHttpRequest, private _tableRest: TableRestService) {
+    super(httpRequest);
+  }
+
+  public getExecutionsTableDataSource(): TableRemoteDataSource<any> {
+    return this.dataSource;
+  }
+
+  public reloadExecutionsTableDataSource(): void {
+    this.dataSource.reload();
+  }
+
+  public getResultColumnItems(): Observable<string[]> {
```

Comment:
Should we specify public modifier?
By default if there is no private or protected modifier, typescript interprets it like public

Reply:
Yes let's discuss it